Sources/ApolloCodegenLib/FileGenerators/OperationManifestFileGenerator.swift

```diff
@@ -57,8 +57,22 @@ struct OperationManifestFileGenerator {
   func generate(fileManager: ApolloFileManager = .default) throws {
     let rendered: String = try template.render(operations: operationManifest)
 
+    var manifestPath = config.output.operationManifest.unsafelyUnwrapped.path
+    let relativePrefix = "./"
+      
+    // if path begins with './' the path should be relative to the config.rootURL
+    if manifestPath.hasPrefix(relativePrefix) {
+      let fileURL = URL(fileURLWithPath: String(manifestPath.dropFirst(relativePrefix.count)), relativeTo: config.rootURL)
+      manifestPath = fileURL
+          .resolvingSymlinksInPath()
+          .appendingPathExtension("json")
+          .path
+    } else {
+      manifestPath.append(".json")
```

Comment:
Do we care whether they've already added `.json` to their `path` value? I suppose it's not going to cause this to stop working, it'll just result in a file ending in `.json.json`.

Reply:
Good point, I will add a check for an existing suffix of `.json` to not duplicate it, won't cover other potential suffixes but we also can't predicate what someone may be using.